dxr/cli/serve.py

```diff
@@ -12,7 +13,7 @@
         flag_value='0.0.0.0',
         help='Serve on all interfaces.  Equivalent to --host 0.0.0.0')
 @option('--host', '-h',
-        default='localhost',
+        default=DXR_DEFAULT_ELASTICSEARCH_HOST,
```

Comment:
This is supposed to be the address the web serve itself listens on, not the ES host.